### PR TITLE
Feature/pdct 1055 frontend create family enable the user to select the corpus

### PR DESCRIPTION
**Pull request by katybaulch**
# What's changed

- Allow the user to select the desired corpus on family create
- Updated tests

## Proposed version

Please select the option below that is most relevant from the list below. This
will be used to generate the next tag version name during auto-tagging.

- [ ] Skip auto-tagging
- [ ] Patch
- [x] Minor version
- [ ] Major version

Visit the [Semver website](https://semver.org/#summary) to understand the
difference between `MAJOR`, `MINOR`, and `PATCH` versions.

Notes:

- If none of these options are selected, auto-tagging will fail
- Where multiple options are selected, the most senior option ticked will be
  used -- e.g. Major > Minor > Patch
- If you are selecting the version in the list above using the textbox, make
  sure your selected option is marked `[x]` with no spaces in between the
  brackets and the `x`

### CODE REVIEW

File: src/interfaces/Config.ts [hunk 1]
```diff
@@ -13,16 +13,49 @@ export interface IConfigGeography {
...
 interface IConfigMeta {
   allow_any?: boolean
   allow_blanks: boolean
   allowed_values: string[]
 }
 
-export interface IConfigLanguageSorted extends OptionBase {
-  value: string
-  label: string
+export interface IConfigTaxonomyCCLW {
```
Comment (diversemix) [inline src/interfaces/Config.ts:36]: Wondering how we are going to manage this when we have many more corpora :thinking:

Comment (katybaulch) [reply src/interfaces/Config.ts:36]: You and me both 🤔

**Review by diversemix — APPROVED** [review]
lgtm - probably worth @PatFawbertMills having a gander too

**Review by PatFawbertMills — APPROVED** [review]
LGTM nice work

**Pull request merged**